docs/source/changelog.rst

```diff
@@ -17,6 +17,7 @@ Changelog
         * Added FAQ (:pr:`620`)
     * Testing Changes
         * Add trigger test for premium primitives (:pr:`660`)
+        * Add release tests premium primitives (:pr:`669`)
```

Comment:
Let's combine this line with the above and say "Add triggers for premium primitive tests"

Reply:
Fixed